docs/analyzer/index.md

```diff
@@ -2,27 +2,159 @@
+download the package and the `en_core_web_lg` spaCy model:
+
+```sh
+pip install presidio-analyzer
+python -m spacy download en_core_web_lg
+```
+
+### Using Docker
+
+> This requires Docker to be installed. [Download Docker](https://docs.docker.com/get-docker/).
+
+```sh
+# Download image from Dockerhub
+docker pull mcr.microsoft.com/presidio-analyzer
 
-- Running in Python.
-- Running using Docker.
-- Creating PII recognizers.
-- [Supporting additional languages and ML models](languages.md)
-- [Outputing the analyzer decision process](decision_process.md)
-- API reference.
+# Run the container with the default port
+docker run -d -p 5001:5001 mcr.microsoft.com/presidio-analyzer:latest
+```
+
+### From source
+
+First, clone the Presidio repo. [See here for instructions](../installation.md#install-from-source).
+
+Then, build the presidio-analyzer container:
+
+```sh
+cd presidio-analyzer
+docker build . -t presidio/presidio-analyzer
+```
+
+## Getting started
 
 ### Running in Python
 
-### Running using Docker
+Once the Presidio-analyzer package is installed, run this simple analysis script:
+
+```python
+from presidio_analyzer import AnalyzerEngine
+
+# Set up the engine, loads the NLP module (spaCy model by default) and other PII recognizers
+analyzer = AnalyzerEngine()
+
+# Call analyzer to get results
+results = analyzer.analyze(text="My phone number is 212-555-5555",
+                           entities=["PHONE_NUMBER"],
+                           language='en')
+print(results)
+
+```
+
+### Running Presidio as an HTTP server
+
+You can run presidio analyzer as an http server using either python runtime or using a docker container.
+
+#### Using docker container
+
+```sh
+cd presidio-analyzer
+docker run -p 5001:5001 presidio-analyzer 
+```
+
+#### Using python runtime
+
+> This requires the Presidio Github repository to be cloned.
+
+```sh
+cd presidio-analyzer
+python app.py
+curl -d '{"text":"John Smith drivers license is AC432223", "language":"en"}' -H "Content-Type: application/json" -X POST http://localhost:3000/analyze
+```
+
+## Creating PII recognizers
+
+- For a tutorial on adding new PII recognizers, [click here](adding_recognizers.md).
+- Fpr best practices for developing new recognizers, [click here](developing_recognizers.md).
+- For multi-language support, [click here](languages.md).
+
+## Outputting the analyzer decision process
+
+Presidio analyzer has a built in mechanism for tracing each decision made. This can be useful when attempting to understand a specific PII detection. For more info, see the [decision process](decision_process.md) documentation.
+
+## Supported entities
+
+For a list of supported entities, [click here](../supported_entities.md).
+
+## API reference
+
+`/analyze`
+
+Analyzes a text. Method: `POST`
+
+Parameters
+
+| Name | Type | Optional | Description|
+| --- | --- | ---| ---|
+| text|string|no|the text to analyze|
+| language|string|no|2 characters of the desired language. E.g en, de|
```

Comment:
maybe add the format we are using for the language?

Reply:
added